### PR TITLE
Consolidate content on the Azure Monitor Metrics doc page

### PR DESCRIPTION
**Pull request by alaudazzi**
## Proposed commit message

On the [Monitor metrics](https://docs.elastic.co/integrations/azure_metrics/monitor), this PR:

- Removes duplicate and confusing content related to Kibana configurations, authentication, and costs. 
- Replaces this content with references to generic instructions clearly documented on the [Azure Resource Metrics](https://docs.elastic.co/integrations/azure_metrics#setup) page.
- Adds the `Requirements` and `Setup` sections to comply with the [documentation guidelines](https://github.com/elastic/integrations/blob/main/docs/documentation_guidelines.md).

## Related issues

- https://github.com/elastic/integration-docs/issues/332
- https://github.com/elastic/integration-docs/issues/324

## Screenshots

image

image

### CODE REVIEW

File: packages/azure_metrics/_dev/build/docs/monitor.md [hunk 1]
```diff
@@ -7,47 +7,16 @@ The azure monitor metrics are numerical values that describe some aspect of a sy
 The Azure Resource Metrics will periodically retrieve the azure monitor metrics using the Azure REST APIs as MetricList.
 Additional azure API calls will be executed in order to retrieve information regarding the resources targeted by the user.
```
Comment (zmoog) [inline packages/azure_metrics/_dev/build/docs/monitor.md:8]:
```suggestion
The Azure Resource Metrics will periodically retrieve the Azure Monitor metrics using the Azure REST APIs as MetricList.
Additional Azure API calls will be executed in order to retrieve information regarding the resources targeted by the user.
```

**Review by zmoog — APPROVED** [review]
LGTM

I added minor changes to capitalization (it's a long fight; we need to start somewhere).

**Comment by muthu-mps** [comment]
#### Overall change looks good. 
Few comments,
- Can we make the reference link to be inserted. Instead of showing the entire URL.
- nit: Please add dot at the end of the line to make it consistent.

Screenshot 2024-04-17 at 7 24 24 PM

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**